lib/adobexd.js

```diff
@@ -23,6 +23,13 @@ async function getArtboardFiles(outputDir) {
   return artboardObjects;
 }
 
+async function getTextStyles(outputDir) {
+  const source = path.join(outputDir, 'resources', 'graphics', 'graphicContent.agc');
```

Comment:
are document assets (text styles, components) always located here?

Reply:
I suppose, that file is a collection of every resource in the XD file (artboards, documentLibrary...)